stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -545,6 +580,10 @@
     (asserts! (check-pox-lock-period num-cycles)
               (err ERR_STACKING_INVALID_LOCK_PERIOD))
 
+    ;; stacking must not happen during prepare phase
+    (asserts! (not (check-prepare-phase burn-block-height))
```

Comment:
Please revert this -- it has not been decided yet that stacking will be disabled in the prepare phase

Reply:
Yes. Reverting all the work here to un-do rebasing from #4094.